policy/modules/admin/logrotate.te

```diff
@@ -150,7 +150,8 @@ optional_policy(`
 
 optional_policy(`
 	apache_read_config(logrotate_t)
-	apache_domtrans(logrotate_t)
+	apache_ctl_domtrans(logrotate_t)
+	apache_ctl_domtrans_from_httpd(logrotate_t)
```

Comment:
This should be a non-`from` interface call from the apache module. The `from` interfaces are intended for users to use without having to modify the policy sources.